examples/gateway/credit_card_signup_non_recurring.php

```diff
@@ -0,0 +1,132 @@
+    $shippingAddress->setAddress('123 Example Street');
+    $shippingAddress->setCity('Miami');
+    $shippingAddress->setState('FL');
+    $shippingAddress->setPostalCode('33000');
+    $shippingAddress->setPhone('1000000000');
+    $creditCardSignup->setShippingAddress($shippingAddress);
+
+    //subscription schedule
+    //you need to send "non_recurring" flag set to true OR subscription_schedule, but not both
+
+    //this is an example of non-recurring signup
+    $creditCardSignup->setIsNonRecurring(true);
+
+    /**
+     * User request details
+     */
+    $request = new \VendoSdk\Gateway\Request\Details\Request();
+    $request->setIpAddress($_SERVER['REMOTE_ADDR'] ?? '127.0.0.1');//you must pass a valid IPv4 address
+    $request->setBrowserUserAgent($_SERVER['HTTP_USER_AGENT'] ?? null);
+    $creditCardSignup->setRequestDetails($request);
+
+    $response = $creditCardSignup->postRequest();
+
+    echo "\n\nRESULT BELOW\n";
+    if ($response->getStatus() == \VendoSdk\Vendo::GATEWAY_STATUS_OK) {
+        echo "The transactions was successfully processed. Vendo's Transaction ID is: " . $response->getTransactionDetails()->getId();
+        echo "\nThe credit card payment Auth Code is: " . $response->getCreditCardPaymentResult()->getAuthCode();
+        echo "\nThe Payment Details Token is: ". $response->getPaymentToken();
+        echo "\nYou must save the payment details token if you need or want to process future recurring billing or one-clicks\n";
+        echo "\nThis is your transaction reference (the one you set it in the request): " . $response->getExternalReferences()->getTransactionReference();
+    } elseif ($response->getStatus() == \VendoSdk\Vendo::GATEWAY_STATUS_NOT_OK) {
+        echo "The transaction failed.";
+        echo "\nError message: " . $response->getErrorMessage();
+        echo "\nError code: " . $response->getErrorCode();
+    } elseif ($response->getStatus() == \VendoSdk\Vendo::GATEWAY_STATUS_VERIFICATION_REQUIRED) {
+        echo "The transaction must be verified";
+        echo "\nYou MUST :";
+        echo "\n   1. Save the payment token: " . $response->getPaymentToken();
+        echo "\n   2. Redirect the user to the verification URL: " . $response->getResultDetails()->getVerificationUrl();
+        echo "\nthe user will verify his payment details, then he will be redirected to the Success URL that's configured in your account at Vendo's back office.";
+        echo "\nwhen the user comes back you need to post the request to vendo again, you can use the TokenPayment class.";
+    }
+    echo "\n\n\n";
+
+
+} catch (\VendoSdk\Exception $exception) {
+    die ('An error occurred when processing your API request. Error message: ' . $exception->getMessage());
+} catch (\GuzzleHttp\Exception\GuzzleException $e) {
+    die ('An error occurred when processing the HTTP request. Error message: ' . $e->getMessage());
+}
+
```

Comment:
Instead of adding a whole new example, could we just add `$creditCardSignup->setIsNonRecurring(true/false);` to the base example?

Reply:
one-liner moved to credit_card_payment